src/copycutter/text_question.py

```diff
@@ -349,41 +372,40 @@ def call_cookie_template(self) -> None:
                 if select.value[-1] is not None:
                     # Again, while choosing defaults checks if the select-field is represented
                     # as a `dict`
-                    context[str(select.value[2])] = select.value[-1][
-                        select._input._options[1][0]
-                    ]
+                    value = select.value[-1][list(select._input._options)[1][0]]
+                    context[str(select.value[2])] = value
                 else:
-                    context[str(select.value[2])] = select._input._options[1][0]
+                    context[str(select.value[2])] = list(select._input._options)[1][0]
         # TODO: Allow this to generalize to other `cookiecutter` templates other than
         # `cookie` in a more structured manner
-        if not os.path.isdir("./tmp"):
-            subprocess.run(["mkdir", "tmp"])
+        if not Path("tmp").is_dir():
+            Path("tmp").mkdir()
         try:
             cookiecutter(
                 template=str(self._template),
                 no_input=True,
-                output_dir=os.path.expanduser("./tmp"),
+                output_dir="tmp",
                 extra_context=context,
             )
         except OutputDirExistsException:
-            subprocess.run(["rm", "-rf", "tmp"])
-            subprocess.run(["mkdir", "tmp"])
+            shutil.rmtree("tmp")
+            Path("tmp").mkdir()
```

Comment:
Spinning up a new process is _very_ expensive. Best to stay in Python.

Small chance that rmtree doesn't support removing a dir with weird permissions, but we can fix that with onerror/onexc if we have to.